src/ripple/overlay/PeerSet.h

```diff
@@ -60,22 +75,14 @@ class PeerSet
 
     bool isActive ();
 
+    // ???
```

Comment:
Hmm...I must have disconnected the question marks from the function it referred to when I moved some members to be protected instead of public. Since I can't remember which one I'll just remove the comment.